app/styles/onboarding-card.module.css

```diff
@@ -77,4 +91,16 @@
         width: 2.8rem;
         height: 2.5rem;
     }
+
+    .btn-generateUsername{
+        width: 5rem;
+        font-size: 12px;
+    }
+}
+
+@media (max-width: 480px) {
+    .onboarding-card{
+        max-width: 90%;
+        padding: 3.5rem 1rem;
```

Comment:
Why you added here again `padding: 3.5rem 1rem;`
already here: https://github.com/Real-Dev-Squad/website-www/pull/648/files#diff-bb9f4c5344f6ff1505af2885cdb84f11eb9ecb1d578d25ee809b6dba3cb75c30R67

Reply:
To make it more aligned in different screen sizes